.github/workflows/pull-request.yml

```diff
@@ -22,6 +22,13 @@ jobs:
       - name: Build Project
         run: pnpm build
 
+      - name: Install Dependencies for API
```

Comment:
Step added for install dependencies for the API and have the build process complete.